### DISCUSSION TITLE
Need OpenWebUI desktop client

### DISCUSSION THREAD

**Discussion by chllei** [open-webui/open-webui#8099] [Ideas]
‌‌‌Hello development team, I really like Open WebUI that you have developed. I was wondering if it's possible to release a corresponding desktop client so that I can use it independently. Thanks very much.

**Comment by DominikStarke** [comment]
I'm currently toying around with the idea of a more general desktop client: https://github.com/DominikStarke/ogre/

It's not really full-featured yet.

**Comment by m8dhouse** [comment]
We created a MAC App (Swift). If people are interested I'll try to clean up the code a bit and make it available for public use somewhere end of Jan.

![Screenshot 2024-12-28 at 12 02 06](https://github.com/user-attachments/assets/ae6a56a6-b87b-45d7-b535-4e41979fb187)
![Screenshot 2024-12-28 at 12 02 18](https://github.com/user-attachments/assets/16d84c3c-7c55-4c9d-9eaa-429708e59ad4)

It uses keyboard shortcut, you can define 4 customs prompts with what model you want to use, and then uses the URL params to push to your open web UI install and start the chat. If you selected a text first and then launch the app then that text is already ready in the context window.

**Reply by jfbloom22** [reply]
This is pretty cool!

**Reply by richtong** [reply]
Amazing. How do we try it and help.

**Reply by richtong** [reply]
Amazing. How do we try it and help.

**Reply by adlanarifzr** [reply]
Is this ready? Where can we download?
